package.json

```diff
@@ -157,6 +157,8 @@
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
     "mocha": "^5.0.0",
+    "mocha-env-reporter": "^3.0.0",
+    "mocha-junit-reporter": "^1.18.0",
```

Comment:
Also FWIW, Circle can parse the "xunit" format that is emitted by mocha's [native/bundled xunit reporter](https://mochajs.org/#xunit) which could save having to add another dev dependency

Reply:
Oh, nice! I’ll give that a shot.

Reply:
It looks like with `xunit` there's not a way to configure the output file through the environment, whereas `mocha-junit-reporter` supports this.

Reply:
Makes sense, not worth fighting xunit here IMO :) My guess is that they will point you to one of the community reporters (like [xunit-file](https://www.npmjs.com/package/xunit-file)) that provide support for env var defined output file, at which point we're right back to what the junit reporter already gives us

Reply:
Heh, possibly. I have a soft spot for Mocha, especially after some recent unpleasant experiences with Jest doing unexpected things with setup. However I do find the way Mocha handles reporters and configuration to be frustrating, fiddly, and to require more effort than it should. I _was_ happy with some upcoming improvements to the config which they mentioned in mochajs/mocha#3612. 🤞

Reply:
I'm a big fan of Mocha too. I've only worked with Jest in one project, but I do hear a lot of good things about it. What sort of setup issues have you seen with Jest?

Reply:
I hit these issues:

- facebook/jest#2713 (it won't be fixed in Jasmine so it's waiting for Jest Circus)
- jasmine/jasmine#577
- jasmine/jasmine#529